recipes/chemem/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set version = "0.0.4" %}
+{% set name = "chemem" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+source:
+  url: https://github.com/SweeneyAaron/ChemEM/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 397bbacc65b40b46b194033388c898e082ffe038c7112b9309012d0afb365662
+build:
+  skip: true  # [win]
+  skip: True  # [py<311]
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  entry_points:
+    - chemem = ChemEM.chemem:main
+    - chemem.test = ChemEM.chemem:test
+    - chemem.protonate = ChemEM.chemem:protonate
+requirements:
+  host:
+    - python
+    - setuptools
+    - pip
+    - openff-toolkit
+  run:
+    - python
+    - numpy
+    - scipy 
+    - dimorphite-dl >=1.3.2
+    - spyrmsd >=0.8.0
+    - rdkit
+    - openff-toolkit
+    - pdbfixer >=1.11
+    - mrcfile >=1.5.0
+    - pathos
+    - biopython >=1.83
+    - scikit-image
+test:
+  requires:
+    - python
+  imports:
+    - ChemEM
+about:
+  home: "https://chemem.topf-group.com/chemem"
```

Comment:
This is a 404 and I cannot see the custom license to know if it is an OpenSource approved or not.